web/concrete/src/Asset/JavascriptAsset.php

```diff
@@ -1,80 +1,95 @@
 <?php
 namespace Concrete\Core\Asset;
+
 use HtmlObject\Element;
 use Config;
 
-class JavascriptAsset extends Asset {
+class JavascriptAsset extends Asset
+{
+
+    protected $assetSupportsMinification = true;
+    protected $assetSupportsCombination = true;
 
-	protected $assetSupportsMinification = true;
-	protected $assetSupportsCombination = true;
+    protected $scriptType = 'text/javascript';
```

Comment:
Well, that would work (aside from the fact that the asset might not be inlined, that this commit also added the possibility of having attributes added to the script tag, and that the AssetType would need to be different since javascript-inline is already taken),

but honestly, I don't quite grab what's wrong with the one I submitted, since that really does not make a big difference here. (Well ok, I can see a little of how maintaining the JS class could break the template class though, but I thought about less code duplication here). 

let me know (but that would be for tomorow, it's already 4am here :()